airflow/providers/google/cloud/transfers/sftp_to_gcs.py

```diff
@@ -106,6 +113,9 @@ def __init__(
         self.move_object = move_object
         self.impersonation_chain = impersonation_chain
         self.sftp_prefetch = sftp_prefetch
+        self.use_stream = use_stream
+        self.stream_chunk_size = stream_chunk_size
+        self.stream_wrapper = None
```

Comment:
Seems like this has the wrong variable name - the doc and later usage say source_stream_wrapper